concurrent/src/main/java/io/airlift/concurrent/VmRuntimeTicker.java

```diff
@@ -118,18 +125,37 @@ private void updateLoop()
             try {
                 Thread.sleep(loopSleepMillis);
 
-                long currentRead = systemTicker.read();
+                // Allocate an object to make sure potential allocation stalls are measured.
+                sleepObjectAllocation = lastRead;
```

Comment:
The field is public and volatile.  I copied this idea from Gil Tene's https://github.com/giltene/jHiccup/blob/master/src/main/java/org/jhiccup/HiccupMeter.java